src/migrations/20200205200615-create-career-applicant-table.ts

```diff
@@ -0,0 +1,31 @@
+import { QueryInterface } from "sequelize";
+import { DataType } from "sequelize-typescript";
+
+export = {
+  up: (queryInterface: QueryInterface) => {
+    return queryInterface.createTable("CareerApplicants", {
```

Comment:
re boludo esto, pero no hay motivo para que haya una entidad en singular y otra en plural, en rails creo que esto seria `CareersApplicants`

Reply:
![image](https://user-images.githubusercontent.com/20487262/74204422-dd795600-4c51-11ea-9b8c-a13383334fed.png)

otra cosa bastante mas importante, corrí las migraciones y no veo `FOREIGN KEY ... REFERENCES ...`, mientras no la tengamos tan clara con sequelize sería paranoico y tiraria describe schema a ver si esto impacta en la BD como uno esperaria

https://medium.com/@andrewoons/how-to-define-sequelize-associations-using-migrations-de4333bf75a7